src/actions/LogActions.tsx

```diff
@@ -76,6 +77,46 @@ export function showSendLogsModal(): ThunkAction<Promise<void>> {
   }
 }
 
+export function showClearLogsModal(): ThunkAction<Promise<void>> {
+  return async (dispatch, getState) => {
+    const state = getState()
+    const { isConnected } = state.network
+    if (!isConnected) return showError(s.strings.network_alert_title)
+
+    Airship.show<string | number | undefined>(bridge => (
```

Comment:
This creates a dangling promise. Totally fine in reality, but we eventually want to enable the lint rule that prohibits this sort of thing. The solution is to either await the promise, or put a `.catch(showError)` on the end.

Reply:
I went with the .catch method because I'm not certain how errors would be caught in a onPress handler if I were to ensure the error propagates up to the event react dispatcher.